spec/rails_helper.rb

```diff
@@ -70,6 +76,7 @@
 Capybara.register_driver :chrome do |app|
   args = %w[no-sandbox disable-gpu window-size=1680,1050]
   args << "headless" unless ENV["NOT_HEADLESS"] == "true"
+  Selenium::WebDriver::Chrome.path = '/usr/bin/google-chrome'
```

Comment:
Does this need to be added? I tried running this locally and ran into an issue. Works without having this here.